### ISSUE TITLE
Text display area fallback suggestion 

### ISSUE THREAD

**Issue by Jake13f** [TechHypno/ClassicFCT#12]
**Suggestion**
Would it be possible to remember the nameplates last position when displaying damage numbers of an enemy that dies and show the final numbers in that location instead of the center?  That way the numbers still show in relatively the same position instead of snapping to the center.  Would add just that much more old style immersion!

> If no nameplate is available (in case the target dies or moves off-screen), text display area can fall back to screen center.

**Comment by TechHypno** [comment]
Getting nameplate positions isn't possible due to blizzard restricting that part of the API.
We can only attach something to a nameplate or not.

**Comment by Jake13f** [comment]
@TechHypno is it possible to get the rendered text’s position instead of the nameplate?

**Comment by TechHypno** [comment]
> @TechHypno is it possible to get the rendered text’s position instead of the nameplate?

No, as long as the inherited position somewhere down the relative chain ends up being a protected frame, anything:GetPoint() will spit out a lua error.

**Comment by TechHypno** [comment]
![image](https://user-images.githubusercontent.com/64381479/166672798-a8b1c60b-8637-431d-ad67-1b08b143df89.png)
target something and run this `/run print(C_NamePlate.GetNamePlateForUnit('target').UnitFrame:GetPoint())`

**Comment by Jake13f** [comment]
@TechHypno dang yeah feel free to close the issue.  It’s really unfortunate unless someone else has some creative idea to still calculate it.  Thanks for the update!

**Comment by TechHypno** [comment]
As of right now there is zero information available for any calculations so unless we buy 51% stake in ATVI its not happening.
